java-to-zod-core/src/main/java/sh/ivan/zod/plugins/PluginParameters.java

```diff
@@ -0,0 +1,320 @@
+package sh.ivan.zod.plugins;
```

Comment:
This class should go into its own submodule, `java-to-zod-plugin-support` or something like that. It shouldn't be packaged with the core. Same with the class above